packages/theme-chalk/src/cascader-panel.scss

```diff
@@ -124,7 +124,8 @@
     text-overflow: ellipsis;
   }
 
-  > .#{$namespace}-radio {
+  > .#{$namespace}-radio,
+  .#{$namespace}-checkbox {
```

Comment:
I think you should separate style for checkbox and radio:

```scss
> .#{$namespace}-checkbox {
  margin-right: 0;
}

> .#{$namespace}-radio {
  margin-right: 0;

  .#{$namespace}-radio__label {
    padding-left: 0;
  }
}
```

Reply:
Thanks for the review.😁